README.md

```diff
@@ -47,6 +47,7 @@ See public/licenses.txt
 * A Ruby manager such as [RVM](http://rvm.io/) or [rbenv](https://github.com/rbenv/rbenv) is strongly recommended.
 * (For shapefile support) access to an [ogre](http://ogre.adc4gis.com) server
 * (For placename completion) a [GeoNames](http://www.geonames.org) account
+* For automatic spatial and temporal extraction from the search text, clone and set up an [EDSC-NLP](https://github.com/mightynimble/edsc-nlp) server
```

Comment:
I'll ping the higher-ups to see what they think. My gut tells me it shouldn't live in a personal account too, but I'm not sure where to put it either. I'll take that action item.